src/EFCore.SqlServer/Storage/Internal/SqlServerStringTypeMapping.cs

```diff
@@ -298,6 +304,18 @@ void AddConcatOperatorIfNeeded()
             {
                 if (builder.Length != 0)
                 {
+                    if (!useOldBehavior2
+                        && !castApplied)
+                    {
+                        builder.Append(" AS ");
+                        if (IsUnicode)
+                        {
+                            builder.Append("N");
+                        }
+                        builder.Append("VARCHAR(MAX))");
```

Comment:
Please don't shout such obscenities at our users... 😉
```suggestion
                            builder.Append("n");
                        }
                        builder.Append("varchar(max))");
```

Reply:
So Query pipeline uses small case when writing store type name but migrations pipeline uses upper case. So where are we heading to?

Reply:
All SQL Server types should be lower-case. I tried to fix everything in 6.0…

Reply:
So are we updating this in 5.0.x? I am fine either way.

Reply:
I just wanted any new 5.0 code to use lower-case so it’s eventually consistent when we merge it into 6.0